### DISCUSSION TITLE
How do you feel about a `GetElementById` for toga?

### DISCUSSION THREAD

**Discussion by proneon267** [beeware/toga#1851] [Ideas]
Hello
Before creating a pull request, I wanted to know how do you feel about a `GetElementById` for toga. Is that something you would approve?

**Comment by freakboy3742** [comment]
Not only would I approve, I already have 😃 

`app.widgets["my_id"]` does widget lookup by ID over the entire app; the same feature is available on a per-window basis.

**Reply by proneon267** [reply]
Thank you!
